lib/codelog/cli.rb

```diff
@@ -4,6 +4,13 @@
 
 module Codelog
   class CLI < Thor
+    map ['--version', '-v'] => :__print_version
+
+    desc '[--version, -v]', 'prints current codelog version'
```

Comment:
Maybe `prints the current version of the gem` or `prints the current  version of codelog` sound better? Either way, I think           this phrase sounds kind of robotic. What do you think?

Reply:
```suggestion
    desc '[--version, -v]', 'prints current of the codelog gem'
```